### ISSUE TITLE
Google Colab

### ISSUE THREAD

**Issue by fabriziogiordano** [fudan-generative-vision/hallo2#18]
It would be great to see a google colab example.

**Comment by mkultra333** [comment]
I agree, a colab would be great, should be easy to set up too.  I started to set one up but ran into a "smudge" bug when I try to download the models, I mention this in another issue. https://github.com/fudan-generative-vision/hallo2/issues/23
